dhis-2/dhis-services/dhis-service-program-rule/src/test/java/org/hisp/dhis/programrule/engine/ProgramRuleEngineDescriptionTest.java

```diff
@@ -79,6 +80,8 @@
 
     private String conditionWithD2DaysBetween = "d2:daysBetween(V{completed_date},V{current_date}) > 0";
 
+    private String conditionWithCalculatedValueRuleVariables = "#{prv1}+#{prv2}>0";
```

Comment:
If this is only used in one test I would inline it. This way I can read the test more easily without having to navigate to this declaration.

Reply:
resolved